src/components/GlobalLeaderboards/TopPositions.tsx

```diff
@@ -0,0 +1,141 @@
+import Pagination from "components/Pagination/Pagination";
+import Table from "components/Table";
+import TableFilterSearch from "components/TableFilterSearch";
+import { useDebounce } from "lib/useDebounce";
+import { useState } from "react";
+import { useLeaderboardContext } from "./Context";
+import { t } from "@lingui/macro";
+import { formatAmount, formatUsd } from "lib/numbers";
+import classnames from "classnames";
+import { TableCell, TableHeader } from "components/Table/types";
+import { USD_DECIMALS, importImage } from "lib/legacy";
+import AddressView from "components/AddressView";
+import { TopPositionsRow, formatDelta } from "domain/synthetics/leaderboards";
+import Tooltip from "components/Tooltip/Tooltip";
+import { getPriceDecimals } from "config/tokens";
+import { BigNumber } from "ethers";
+import { MarketInfo } from "domain/synthetics/markets";
+
+const formatPriceDecimals = (x: BigNumber, market: MarketInfo, chainId: number) => formatAmount(
+  x,
```

Comment:
we trying to avoid using of "x", "a" etc. parameters and variables since we have a lot of different numbers and math and it could be unclear what x means here. 

also, to avoid confusion in V2 we use "usd" and "amount" endings for usd and token amount values. eg. feeUsd / feeAmount.